app/frontend/helpers/NamedError.ts

```diff
@@ -1,8 +1,8 @@
-function NamedError(name, message, showHelp) {
+function NamedError(name, message, showHelp='') {
```

Comment:
all in all, this should probably be rewritten into class form

Reply:
btw. the `showHelp` argument is completely useless, just a forgotten piece code from previous version, we can remove it.